2 - RTB/Documentazione esterna/Piano di Progetto/Piano di Progetto.typ

```diff
@@ -626,41 +905,45 @@ Gli obiettivi dell'ottavo sprint riguardano tutti gli aspetti del progetto, di c
-    [*Membro*], [*Responsabile*], [*Amministratore*],[*Analista*],[*Progettista*], [*Programmatore*], [*Verificatore*],[*Totale*],
-    [Banzato Alessio],   [/],  [3],    [/],    [/],    [/],   [/], [3],
-    [Carraro\ Riccardo], [/],  [/],    [/],    [/],    [3],   [/], [3],
-    [Gardin\ Giovanni],  [/],  [/],    [/],    [3],    [/],   [/], [3],
-    [Nardo\ Silvio],     [4],  [/],    [/],    [/],    [/],   [/], [4],
-    [Oseliero\ Antonio], [/],  [/],    [/],    [/],    [/],   [4], [4],
-    [Todesco Mattia],    [/],  [/],    [/],    [3],    [/],   [/], [3],
-    [Zaccone\ Rosario],  [/],  [/],    [3],    [/],    [/],   [/], [3],
-    [Totale\ ore ruolo], [4],  [3],    [3],    [6],    [3],   [4], [23],
-    [Costo orario],     [30,00], [20,00], [25,00], [25,00], [15,00], [15,00], [],
-    [Costo ruolo],      [120,00], [60,00], [75,00], [150,00], [45,00], [60,00], [510,00]
+    [*Membro*], [*Responsabile*], [*Amministratore*], [*Analista*], [*Progettista*], [*Programmatore*], [*Verificatore*], [*Totale*],
+    [Banzato],     [0],     [3],     [0],     [0],     [0],     [0],     [3],
+    [Carraro],     [0],     [0],     [0],     [0],     [3],     [0],     [3],
+    [Gardin],     [0],     [0],     [0],     [3],     [0],     [0],     [3],
+    [Nardo],     [4],     [0],     [0],     [0],     [0],     [0],     [4],
+    [Oseliero],     [0],     [0],     [0],     [0],     [0],     [4],     [4],
+    [Todesco],     [0],     [0],     [0],     [3],     [0],     [0],     [3],
+    [Zaccone],     [0],     [0],     [3],     [0],     [0],     [0],     [3],
+    [Totale ore],     [4],     [3],     [3],     [6],     [3],     [4],     [23],
+    [Costo ruolo],     [120],     [60],     [75],     [150],     [45],     [60],     [510],
   ),
-  caption: "Preventivo ottavo sprint"
+  caption: "Prospetto del preventivo, sprint 8"
 )
-
-#set text(
-  size: 11pt
+#let data = (
+  ("Responsabile", 4),
+  ("Amministratore", 3),
+  ("Analista", 3),
+  ("Progettista", 6),
+  ("Programmatore", 3),
+  ("Verificatore", 4),
+)
+#figure({
+  roles-legend
+  canvas({
+    import draw: *
+    chart.piechart(..piechart-config, data)}
+  )},
+  caption: "Suddivisione oraria per ruolo, preventivo sprint 8",
+  kind: "chart",
+  supplement: "Grafico"
 )
 
-=== Diagramma di Gantt
-[da inserire]
-
-
-== Nono Sprint dal 01-01-2024 al 08-01-2024
+== Sprint 9 dal 01-01-2024 al 08-01-2024
 
 === Obiettivi prefissati <obiettivi9>
-Gli obiettivi dell'ottavo sprint si incentrano sul completamento delle attività rimaste sospese nell'ottavo sprint, sulla revisione dei documenti e del PoC in vista della valutazione RTB. In particolare gli obiettivi prefissati per questo sprint sono:
+Gli obiettivi dell'ottavo sprint si incentrano sul completamento delle attività rimaste sospese nell'ottavo sprint, sulla revisione dei documenti e del PoC in vista della valutazione RTB. , gli obiettivi prefissati per questo sprint sono:
```

Comment:
in "...valutazione RTB. , gli obiettivi..." c'è una virgola in più e manca la maiuscola in "gli"